src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -213,7 +213,11 @@ protected function finalizeValue($value)
         foreach ($this->children as $name => $child) {
             if (!\array_key_exists($name, $value)) {
                 if ($child->isRequired()) {
-                    $ex = new InvalidConfigurationException(sprintf('The child node "%s" at path "%s" must be configured.', $name, $this->getPath()));
+                    $message = sprintf('The child config "%s" under "%s" must be configured.', $name, $this->getPath());
+                    if ($child->getInfo()) {
+                        $message .= sprintf("\n\n Description: %s", $child->getInfo());
```

Comment:
I'm not sure if the `\n\n` is a problem. It sure does help readability.

Reply:
Having `\n\n` looks ok to me. Maybe we can remove the space before `Description: `, I would even remove the `Description: ` prefix.

Reply:
Another alternative could be

Reply:
Another alternative could be 
```

Reply:
`Expected value for the "%s" option: %s` instead of `Description`?

Reply:
What do you think? Having a one liner as suggested by Nicolas seems good to me.

Reply:
@weaverryan Not convinced?